server/config.go

```diff
@@ -45,6 +46,11 @@ type CacheConfig struct {
 	MaxSize datasize.ByteSize `yaml:"max_size"`
 }
 
+type WorkerConfig struct {
```

Comment:
maybe go-githubapp can provide configuration object(s) for the different dispatcher options? Then servers can reuse the objects (like with baseapp.HTTPConfig)

Then callers can do `githubapp.WithSchedulerFromConfig(config.XXX)`.

Reply:
Hmm, right now, the `QueueAsyncScheduler` is the only one with options that users might adjust. I don't think we want to make the scheduler type itself configurable, as it can have a pretty big impact on application functionality.

Caching is another area where we current defer to end applications for configuration. I'll think about extracting some of these structs, but I'm not sure it will by worthwhile.

Reply:
I think I'm going to leave the config stuct here for now, it feels like it doesn't really fit with the other configuration code currently in `go-githubapp`.